packages/components/src/organisms/Filter/Filter.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Accordion as UIAccordion } from '../../'
+
+import { PropsWithChildren } from 'react'
+
+export type OnFacetChange = (
+  item: { key: string; value: string },
+  type: 'BOOLEAN' | 'RANGE'
+) => void
+
+export interface FilterProps {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId: string
+  /**
+   * Title for the `Filter` component.
+   */
+  title?: string
+  /**
+   * The expanded items from the `Accordion`.
+   */
+  indicesExpanded: Set<number>
+  /**
+   * This function is called when `Accordion` is expanded or collapsed.
+   */
+  onAccordionChange: (index: number) => void
+}
+
+function Filter({
+  testId,
+  title,
+  indicesExpanded,
+  onAccordionChange,
+  children,
+}: PropsWithChildren<FilterProps>) {
+  return (
+    <div data-fs-filter data-testid={testId}>
+      <h2 className="text__title-mini-alt" data-fs-filter-title>
```

Comment:
I think we don't use this kind of classes in components package

Reply:
Changes [here](https://github.com/vtex/faststore/pull/1687/commits/5a112c275706d7c3b866343a5fb1aa03c2cdb421).